features/support/run_helpers.rb

```diff
@@ -63,17 +63,18 @@ def result_has_shell_error? result
 end
 
 
-def run_shell_command command, input
+def run_shell_command command, inputs
   result = OpenStruct.new(command: command, location: Pathname.new(Dir.pwd).basename)
   command = "#{shell_overrides}; #{command} 2>&1"
+  kill = inputs.pop && true if inputs.last == '^C'
```

Comment:
I'm confused how killing works. Can you please add a comment as to what this does? 
Also is the `&& true` necessary? Wouldn't kill just end up being `'^C'` which is truthy anyway.